migrations/post_install/post_install.sql

```diff
@@ -80,18 +80,21 @@ do $$
 
       -- Migration Script to populate jobstatesMap for builds
 
-      create temp table tjsm as WITH cte AS
-      (
-         SELECT "id", "projectId", "resourceId", "createdAt", "statusCode", "subscriptionId", "updatedAt", ROW_NUMBER() OVER (PARTITION BY "projectId", "resourceId", "statusCode" ORDER BY "createdAt" desc)
-      AS rn FROM builds WHERE "statusCode" in (4002,4003)
+      create temp table tjsm as WITH cte AS (
+        SELECT "id", "projectId", "resourceId", "createdAt", "statusCode", "subscriptionId", "updatedAt",
+          ROW_NUMBER() OVER (PARTITION BY "projectId", "resourceId", "statusCode" ORDER BY "createdAt" desc)
+          AS rn
+          FROM builds
+          WHERE "statusCode" in (4002,4003)
       )
       SELECT "id", "projectId" as pid, "resourceId" as rid, "createdAt", "statusCode", "subscriptionId", "updatedAt"
       FROM cte
       WHERE rn = 1;
 
-      create temp table rtjsm as WITH cte AS
-      (
-         SELECT "id", "projectId", "createdAt", "createdBy", "updatedBy", "name", "typeCode", "isJob", "lastVersionName", "lastVersionNumber" FROM resources WHERE "isJob" = true and "typeCode" in (2010,2000,2005,2007,2012,2011,2009)
```

Comment:
We don't need this array of resource type codes since, we are using "isJob" = true. This will make sure that all the pipeline jobs are returned.